### PR TITLE
Sentiment Analysis with BERT and RoBERTa

### PR DESCRIPTION
**Pull request by sagar27sahu**
# Pull Request for DL-Simplified 💡
This project implements sentiment analysis using state-of-the-art transformer models, BERT and RoBERTa. The aim is to classify text data into positive and negative sentiments by fine-tuning pre-trained models on a custom dataset. The project includes data preprocessing, model training, and evaluation metrics such as accuracy, precision, recall, and F1 score, allowing for a comparative analysis of the two models' performance in sentiment classification tasks.

## Issue Title : 

- **Info about the related issue (Aim of the project)** : 
- **Name:** <sagar kumar sahu>
- **GitHub ID:** <sagar27sahu>
- **Email ID:** <sagar2004sahu01@gmail.com>
- **Idenitfy yourself: (Mention in which program you are contributing in. Eg. For a JWOC 2022 participant it's, `JWOC Participant`)** <hacktoberfest and gssoc-ext 2024>

Closes: #588 

### Describe the add-ons or changes you've made 📃

i have made sentiment analysis using Bert and Robert

## Type of change ☑️

What sort of change have you made:
<!--
Example how to mark a checkbox:-
 enhancing the code of sentiment analysis.

## How Has This Been Tested? ⚙️

Describe how it has been tested
in google colab also requried api key for this.

## Checklist: ☑️
<!--
Example how to mark a checkbox:-

 My code follows the guidelines of this project.
I have performed a self-review of my own code.

- [ ] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Hi @sagar27sahu 

1. As this is an enhancement work, you don't have to create a new project folder. Check out the existing project folder and add your `ipynb` files there with the implemented model names.
2. Do not use `.py` files, use `ipynb` files only.
3. Update the `README.md` in the existing project folder as per your contributions.

Hope this is clear to you.

**Comment by sagar27sahu** [comment]
@abhisheks008 i have resolved the issue,please check and please consider increasing label1 to label2

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
1. Remove the unnecessary files (`.DS_Store`, `.idea` and so on).
2. You were asked to not to change/update the existing project `.ipynb` file, still you have updated that.
3. No need to create a separate `README.md`. In the existing `README.md` mention your implemented models' details, as per the existing points of the README.
4. Update the file name as `requirements.txt`, it's not a `.docx` file.

Please have a look at the requested changes for this pull request.
@sagar27sahu

**Comment by abhisheks008** [comment]
`.DS_Store` files are still there. Please remove them.
